src/app/modules/users/user.entity.ts

```diff
@@ -26,6 +26,13 @@ export class UserEntity {
   })
     email: string;
 
+  @Column ({
```

Comment:
Wrong implementation. ClassicAuthEntity contain status field. You must use the same approach in User entity.